### ISSUE TITLE
On the entire page navigation doesn't have role

### ISSUE THREAD

**Issue by rseyum** [rseyum/github-issue-templates#9]
**Describe the bug**
on the entire page, navigation doesn't contain role attributes which can affect screen readers.

**To Reproduce**
1. Use any screen reader and explore the app
2. you will notice the navigation summary is not announced
3. got to HTML and add role="navigation" aria-label ""
5. Check the result

**Expected behavior**
When we start the screen reader should be able to announce navigation.

**Screenshots**
N/A

**Desktop (please complete the following information):**
N/A
**Smartphone (please complete the following information):**
N/A

**Additional context**
Missing this feature can affect visually impaired or disabled people negatively it might not be clear where they are.
